### PR TITLE
Cleanup after cwr removal

### PR DESCRIPTION
**Pull request by ktsakalozos**

### CODE REVIEW

**Comment by johnsca** [comment]
This should fix #7

File: lib/CIGWServer.py [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
-
-from os import path
 from io import open
 from json import dumps
 from flask import Flask, request
 from jenkins import Jenkins
 from controller.helpers import get_controllers
 
 
+sys.path.append('lib')
```
Comment (kwmonroe) [inline lib/CIGWServer.py]: i don't think this will work.  you need to append 'lib' prior to importing controller.helpers, right?

Comment (kwmonroe) [inline lib/CIGWServer.py]: Oh wait, nm.  This appends $CHARM_DIR/lib.  controller.helpers lives in pwd, which confused me because pwd is 'lib' too.

**Comment by kwmonroe** [comment]
verified fixes #7

**Pull request merged**